### PR TITLE
mass number matching

### PR DESCRIPTION
**Pull request by snschune**
refs #300

### CODE REVIEW

**Comment by snschune** [comment]
I forgot that we currently match only by `Z` if only one isotope of an element is present. 
This is somewhat confusing behavior and I decided to remove it.

**Comment by snschune** [comment]
Actually, shouldn't we do the mass number range by variable in the rasterizer? If it's not supplied it's set to a vector of `1/2` of the right length.

File: include/userobjects/MyTRIMRasterizer.h [hunk 1]
```diff
@@ -58,13 +58,14 @@ class MyTRIMRasterizer : public ElementUserObject
...
     AveragedData(unsigned int nvars = 0)
-      : _elements(nvars, 0.0), _Z(nvars, 0.0), _M(nvars, 0.0), _site_volume(0.0)
+      : _elements(nvars, 0.0), _Z(nvars, 0.0), _M(nvars, 0.0), _M_tol(nvars, 0.5), _site_volume(0.0)
     {
     }
 
     std::vector<Real> _elements;
     std::vector<Real> _Z;
     std::vector<Real> _M;
+    std::vector<Real> _M_tol;
```
Comment (dschwen) [inline include/userobjects/MyTRIMRasterizer.h]: This dos not seem right. `AveragedData` is stored in a map with an entry for _every_ active element. I don't think we'll need spatially varying mass tolerance.

Comment (snschune) [reply include/userobjects/MyTRIMRasterizer.h]: The same is should then be true for `_M` and `_Z`.

Comment (dschwen) [reply include/userobjects/MyTRIMRasterizer.h]: Yes... I think you're right!

File: include/userobjects/MyTRIMRasterizer.h [hunk 2]
```diff
@@ -67,7 +67,7 @@ class MyTRIMRasterizer : public ElementUserObject
   struct PKAParameters
   {
     /// masses charges (m_i, Z_i) of the matrix elements
```
Comment (dschwen) [inline include/userobjects/MyTRIMRasterizer.h]: This comment should be updated

**Pull request merged**